docs/getting-started.md

```diff
@@ -84,7 +84,7 @@ We hebben al een project voor je gemaakt die je kan gebruiken: [BAG-Bevragen-pos
 
 ### API key
 Om de API te kunnen bevragen is een API key nodig. Deze moet je bij het request opnemen in request header "X-Api-Key".
-Vraag een API key aan voor de [testomgeving](https://formulieren.kadaster.nl/aanvraag_bag_api_huidige_bevragingen_test){:target="_blank"}.
+[Vraag een API key voor de productieomgeving aan](https://formulieren.kadaster.nl/aanvraag_bag_api_huidige_bevragingen_productie){:target="_blank"}. 
 
 ### URL
 De API is te benaderen via de volgende url: https://api.bag.acceptatie.kadaster.nl/esd/huidigebevragingen/v1.
```

Comment:
Zo te zien wil je dat er in de productie omgeving wordt getest. Is het dan nog nodig om dit te laten staan?